content/english/python-interactive/activity-4.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Activity 4 - Interacting with the Customer"
+date: 2019-07-25T13:24:17-07:00
+weight: 3
+draft: false
+translationKey: "python-INTERACTIVE-activity-4"
+---
+## Interacting with the Customer
+Let's continue interacting with the customer and get their order! In the `main()` function, the next thing we need to do is create an empty list stored in the variable `lstOrderItems` so that any items they will eventually order are stored here.
+
+Now let's use our knowledge of if-else statements and while loops to receive the customer's order. If the customer responded No to the question asking them whether they would like to order anything from the menu, then print out a Goodbye statement. Otherwise, ask them to input the first item and quantity they would like to order with a specified format such as `"(Example order item: Spring Rolls-2)"` provided in the prompt.   
+
+We've got the first item the customer wants to order! Well, that's what we think at least... However, we don't actually know what the customer entered into the `input()` function, they might have just entered `"Baa Baa Baa Ba ba banana"` from the Minions movie. Or they might have entered a valid menu item, but it's not in the format you specified. They might have written `"Spring Rolls : 2"` or `"Spring Rolls -2`, neither of which is a valid input! Or they might have even given you a NEGATIVE quantity, point being, we have no clue what the customer might have entered, and we need to protect the customer and our program against this invalid input.
+
+So let's write a function that will check any user's order item input and call it `orderItemChecking(item)`. Notice how we gave it a parameter? That signals we need to pass in the order item as an argument to this function from the `main()`. In this function we're going to want to safeguard against all of the possible pitfalls mentioned in the previous paragraph. 
+
+Hint: Use if statements to catch incorrect input.
+
+Hint: Return a boolean value, either `True` if the input is valid or `False` if the input is invalid.
+
+Hint: A helpful string method to analyze the orderItemInput would be str.split()... what should we split by?
```

Comment:
Haven't used method a lot previous, should we change this to function?

Reply:
```suggestion
Hint: A helpful string method to analyze the orderItemInput would be `str.split()`... what should we split by?
```